lanl/docker-compose/ochami-krakend-ce.yml

```diff
@@ -1,20 +1,43 @@
 version: '3.7'
 
 services:
-  
+  step-ca:
+    hostname: SI.ca
+    container_name: SI.ca
+    image: ghcr.io/openchami/local-ca:acme
+    ports: 
+      - "443:443"
+    healthcheck: 
+      test: ["CMD-SHELL", "step", "ca", "health", "--ca-url", "https://SI.ca", "--root", "$(step path)/certs/root_ca.crt"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+      start_period: 10s
+    networks:
+      - internal
+    volumes:
+      - ./configs/step-ca:/mnt
   krakend-ce:
-    container_name: krakend-ce
-    image: devopsfaith/krakend
+    hostname: SI.krakend
+    container_name: SI.krakend
+    image: ghcr.io/openchami/krakend:acme
     environment:
       - KRAKEND_CONFIG=/etc/krakend/krakend.json
       - KRAKEND_PORT=8080
-    hostname: krakend-ce
+    healthcheck:
+      test: ["CMD-SHELL", "CURL_CA_BUNDLE=/root/.acme.sh/SI.krakend_ecc/SI.krakend.cer", "curl", "-f", "https:/SI.krakend:8080/__health"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+      start_period: 10s
     depends_on:
       - bss
       - smd
+      - step-ca
     ports:
       - "8080:8080"
     networks:
       - internal
     volumes:
-      - ./configs/krakend.json:/etc/krakend/krakend.json:ro
+      - ./configs/krakend.json:/etc/krakend.json:ro
+      - ./configs/step-ca/config/defaults.json:/mnt/defaults.json:ro
```

Comment:
is defaults.json intended to be present before launching the docker compose or will it be created by the step-ca container?  We should indicate either way with comment(s) in the file.

Reply:
It's auto generated by launching the step-ca container. I'll add it as a comment. In this case (the krakend container) the `defaults.json` contains the fingerprint that the step cli uses to bootstrap the core CA into the container. Will add a comment.